src/ApolloHooksApolloProvider.re

```diff
@@ -2,4 +2,4 @@
 external make:
   (~client: ApolloClient.generatedApolloClient, ~children: React.element) =>
   React.element =
-  "ApolloProvider";
```

Comment:
It is recommended to have an empty line in each file, otherwise it won't be considered a text file in unix (I think so? 😄 )
@jfrolich  , could you please add empty files in the files that don't have them? There is a setting in `vscode` that can do it for you, if you are using `vscode` of course 😄

Reply:
ok, I didn't have the setting activated.